build.gradle.kts

```diff
@@ -94,6 +97,7 @@ val copyGradleScripts by tasks.registering(Copy::class) {
 
     // local variable required for configuration cache compatibility
     // https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:not_yet_implemented:accessing_top_level_at_execution
+    val buildScanSummaryVersion = buildScanSummaryVersion
     val releaseVersion = releaseVersion
     inputs.property("project.version", releaseVersion)
```

Comment:
Do we also need to declare an input property for the `buildScanSummaryVersion` ?

Reply:
I don't think so, since the [commonComponents](https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/471e8189d42a756847d22ea20070c9afe9f1ec8a/build.gradle.kts#L166) configuration will change

Reply:
I think we should still explicitly declare it since we use the buildScanSummaryVersion in the text-replacement logic.

Reply:
Done in https://github.com/gradle/gradle-enterprise-build-validation-scripts/pull/487/commits/4e1bc772c38dd7e59e3c890a370d257ec391a850